pages/learn/deploy/offline-updates.md

```diff
@@ -0,0 +1,273 @@
+---
+title: Offline updates
+excerpt: An efficient approach for updating previously deployed IoT devices when they’re offline by design.
+---
+
+# Offline updates
```

Comment:
@vipulgupta2048 we need to standardise on the terminology. In the blog we are calling these "offline upgrades".